src/shared/dropdown-wrapper/dropdown-wrapper.styles.tsx

```diff
@@ -18,49 +18,67 @@ export interface DropdownWrapperStyleProps {
 
 export interface ValueLabelStyleProps {
     truncateType?: TruncateType;
+    variant?: DropdownVariantType | undefined;
+}
+
+export interface SelectorStyleProps {
+    variant?: DropdownVariantType;
```

Comment:
for non-HTML attributes, prefix them with `$` to mark them as transient props

please update for the other style interfaces

Reply:
whoops sorrry if it wasn't clear

for component props, there's no need to prefix them with `$`

however for styled components, `$` is needed to mark them as transient, so that extra non-standard attributes are not passed down to the underlying HTML element

Reply:
@qroll i have pushed a commit to remove `$` from the component props.
you can have a look again :)